.lycheeignore

```diff
@@ -6,14 +6,9 @@ https://codepen.io
 
 # Network error: The certificate was not trusted
 https://mkdocs.org/
```

Comment:
By the way, if we want to remove this too, apparently we can just add `www.` to all the `mkdocs.org` links, since https://www.mkdocs.org doesn't have any certificate issues, (or we can wait for https://github.com/mkdocs/mkdocs/issues/1626 to be fixed!)

Reply:
Cool, removed it from the list and changed the links to https://www.mkdocs.org/. Tested locally and seems to work.